money_bot/utils/db_utils.py

```diff
@@ -1,6 +1,22 @@
 from money_bot.utils import models
 
 
+async def is_user_in_db(user_id: int):
+    return bool(await get_user_by_id(user_id))
```

Comment:
Use get_user_by_id is None, I think it's better

Reply:
agree